enhancements/sig-architecture/63-hosted-addon/README.md

```diff
@@ -0,0 +1,276 @@
+- When the label's value is `none`: the manifest will not be deployed in Hosted mode
+
+
+<table class="tg">
+<thead>
+  <tr>
+    <th class="tg-0pky" rowspan="2"><br>Label existence/value</th>
+    <th class="tg-c3ow">Default Mode</th>
+    <th class="tg-c3ow" colspan="2">Hosted Mode</th>
+    <th class="tg-c3ow" rowspan="2"><br>Example</th>
+  </tr>
+  <tr>
+    <th class="tg-c3ow">Managed Cluster</th>
+    <th class="tg-c3ow">Managed Cluster</th>
+    <th class="tg-c3ow">Hosting Cluster</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td class="tg-0pky"><span style="font-weight:400;font-style:normal">nonexist | managed</span></td>
+    <td class="tg-c3ow">Y</td>
+    <td class="tg-c3ow">Y</td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow">CRD owned by the addon</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky"><span style="font-weight:400;font-style:normal">hosting</span></td>
+    <td class="tg-c3ow">Y</td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow">Y</td>
+    <td class="tg-c3ow">deployment</td>
+  </tr>
+  <tr>
+    <td class="tg-0pky"><span style="font-weight:400;font-style:normal">none</span></td>
+    <td class="tg-c3ow">Y</td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+    <td class="tg-c3ow"></td>
+  </tr>
+</tbody>
+</table>
+
+### For addon deployers
+
+Once the managed cluster is imported to the hub in `Hosted` mode, the addon deployers can choose to run addon in Hosted
+mode or Default mode, By default the addon agent will run on the managed cluster(Default mode), the addon deployers can 
+add an annotation `addon.open-cluster-management.io/hosting-cluster-name` for `ManagedClusterAddon` so that the addon 
+agent will be deployed on the certain hosting cluster(Hosted mode), the value of the annotation is the hosting cluster 
+which should:
+- be a managed cluster of the hub as well
+- be the same cluster where the managed cluster klusterlet(registration-agent & work-agent) runs
+
+#### How does the addon agent connect to the managed cluster?
+
+Addon agents deployments will be deployed in the `{ManagedClusterAddon.Spec.InstallNamespace}` namespeace, and the 
+addon deployer is required to provide a secret, which name is `{addon-name}-managed-kubeconfig` and contains the key 
+`kubeconfig` to connect to the managed cluster, in the `{ManagedClusterAddon.Spec.InstallNamespace}` namespace on the 
+hosting cluster.
+
+Some potential ways to provide the secret:
+- The addon deployer asks the managed cluster administrator for a specific permission kubeconfig, then create it on the 
+hosting cluster. Because the kubeconfig is generated by the managed cluster administrator, the addon developer does not 
+need to deploy any service-account, role, rolebinding on the managed cluster, labels these resources 
+`addon.open-cluster-management.io/hosted-manifest-location: none`
+- The addon developer deploys service-account, role, rolebinding on the managed cluster,labels these resources 
```

Comment:
Sorry, this may be confusing. I mean once the addon developer adds the label addon.open-cluster-management.io/hosted-manifest-location: managed or keeps the label non-exist, the addon-framework will deploy the resource to the managed cluster.